src/library_wasmfs.js

```diff
@@ -179,11 +179,20 @@ FS.createPreloadedFile = FS_createPreloadedFile;
     chmod: (path, mode) => {
       return withStackSave(() => {
         var buffer = stringToUTF8OnStack(path);
-        return __wasmfs_chmod(buffer, mode);
+        return FS.handleError(__wasmfs_chmod(buffer, mode));
       });
     },
     // TODO: lchmod
+    lchmod: (path, mode) => {
+      return withStackSave(() => {
+        var buffer = stringToUTF8OnStack(path);
+        return FS.handleError(__wasmfs_lchmod(buffer, mode));
+      });
```

Comment:
```suggestion
      return FS.handleError(withStackSave(() => {
        var buffer = stringToUTF8OnStack(path);
        return __wasmfs_lchmod(buffer, mode));
      }));
```
(the error handling doesn't need to be inside, and it's generally nicer to keep the `withStackSave` on just the code that needs it)

Reply:
This was resolved but I still see the `handleError` inside, when as suggested it would be better on the outside?